runtime/src/bank.rs

```diff
@@ -2764,6 +2765,32 @@ impl Bank {
         }
     }
 
+    #[allow(dead_code)]
+    fn hash_rewards_into_partitions(
+        stake_rewards: &StakeRewards,
+        rewarded_epoch: Epoch,
+        rewards: u64,
+        num_partitions: usize,
+        thread_pool: &ThreadPool,
+    ) -> Vec<StakeRewards> {
+        let seed = rewarded_epoch ^ rewards;
+        let hasher = create_epoch_reward_hasher(seed);
+
+        let mut result = vec![vec![]; num_partitions];
+
+        thread_pool.install(|| {
+            let partitions = stake_rewards
+                .par_iter()
+                .map(|reward| address_to_partition(hasher, num_partitions, &reward.stake_pubkey))
```

Comment:
```suggestion
                .map(|reward| address_to_partition(hasher.clone(), num_partitions, &reward.stake_pubkey))
```

right?

Reply:
Looks like `siphasher` implements Copy traits. Therefore, at the call site, a copy will generated by the compiler. no need to clone.

Reply:
ah. i figured we'd need `H: Hasher + Copy` on `address_to_partition()` to make that work, rather than the `Clone` that's there now

Reply:
yeah. updated.